lib/operations/insert_one.js

```diff
@@ -14,17 +14,17 @@ const Aspect = operation.Aspect;
 
 class InsertOneOperation extends OperationBase {
   constructor(collection, doc, options) {
-    super(options);
+    let finalOptions = Object.assign({}, options);
```

Comment:
Is `InsertOneOperation` now irrelevant because `collection.insertOne` uses `InsertOneModel` and `CrudWriteOperation`?

Reply:
yeah, all the ops using CrudWriteOperation should be removed.